examples/scripts/register_task_plugins.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# This Software (Dioptra) is being made available as a public service by the
+# National Institute of Standards and Technology (NIST), an Agency of the United
+# States Department of Commerce. This software was developed in part by employees of
+# NIST and in part by NIST contractors. Copyright in portions of this software that
+# were developed by NIST contractors has been licensed or assigned to NIST. Pursuant
+# to Title 17 United States Code Section 105, works of NIST employees are not
+# subject to copyright protection in the United States. However, NIST may hold
+# international copyright in software created by its employees and domestic
+# copyright (or licensing rights) in portions of software that were assigned or
+# licensed to NIST. To the extent that NIST holds copyright in this software, it is
+# being made available under the Creative Commons Attribution 4.0 International
+# license (CC BY 4.0). The disclaimers of the CC BY 4.0 license apply to all parts
+# of the software developed or licensed by NIST.
+#
+# ACCESS THE FULL CC BY 4.0 LICENSE HERE:
+# https://creativecommons.org/licenses/by/4.0/legalcode
+
+import os
+from pathlib import Path
+
+import click
+from clients import DioptraClient
+from utils import make_custom_plugins
+
+_CURRENT_DIR = Path(__file__).parent
+_CONTEXT_SETTINGS = dict(
+    help_option_names=["-h", "--help"],
+    show_default=True,
+)
+
+
+class DIRECTORY:
+    def __init__(self, directory):
+        self.directory = directory
+
+
+class URL:
+    def __init__(self, url):
+        self.url = url
+
+
+@click.command(context_settings=_CONTEXT_SETTINGS)
+@click.option(
+    "--plugins-dir",
+    default="./task-plugins",
+    type=DIRECTORY,
+    help="The path to the directory containing the custom task plugin subdirectories.\n",
+)
+@click.option(
+    "--api-url",
+    default="http://localhost",
+    type=URL,
+    help="The url to the Dioptra REST API.\n",
```

Comment:
A new line is not necessary in the help messages.

```suggestion
    help="The url to the Dioptra REST API.",
```